samples/extensions/extended_dynamic_state_3_multisample_rasterization/README.adoc

```diff
@@ -0,0 +1,61 @@
+////
+- Copyright (c) 2023, Mobica Limited
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+////
+
+// Extended dynamic_state3: Rasterization samples
+
+
+== Overview
+
+This sample demonstrates one of the functionalities of VK_EXT_extended_dynamic_state3 related to rasterization samples.
+The extension can be used to dynamically change sampling without any need to restart the application.
```

Comment:
This line is a little misleading. It implies that you can only normally change sampling by restarting an application, which obviously isn't the case.